### PR TITLE
Bp/api calls

### LINKED ISSUE #39: API
on page load, get specific users stored recipe objects
(pictures, titles, favoriting)

### LINKED ISSUE #40: API
have domUpdates.js

### LINKED ISSUE #41: link domUpdates.js in scripts.js
link domUpdates.js in scripts.js

### PR DESCRIPTION
**Pull request by benjaminprat**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
Connects api to user endpoints and fetches data.
Adds domUpdates file.
Displays fetched data on DOM 

## *Instructions for reviewer:*

*How should this be manually tested?*
npm start to both localhosts
open page and click through functionality. 

*What are the relevant tickets?*
closes #39
closes #40 
closes #41

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*
There is probably too much going on in this PR so apologies for the infrequent commits. 
We can eventually move the apiCalls to their own file. 
I think some of this work could go back and be refactored. 

## Checklist:
- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

**Comment by Max9545** [comment]
take a look at passing those HTML to the DOM.updater again as a refactor if we have time

File: src/domUpdates.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+let domUpdates = {
```
Comment (gaj23) [inline src/domUpdates.js:1]: Awesome! We were really really close to getting things going on Saturday. Thanks for all your hard work.

File: src/index.html [hunk 2]
```diff
@@ -14,21 +14,23 @@ <h1>What's Cookin'</h1>
...
         <button id='home-button' class='home navButton'>Home</button>
-        <label for='view-favorites-button' class='hidden'>View favorites</label>
-        <button id='view-favorites-button' class=' view-favorites navButton'>View Favorites</button>
-        <!-- <form class='search-bar'>
+        <label for='view-favorites-button' class='hidden'>My Favorites</label>
+        <button id='view-favorites-button' class=' view-favorites navButton'>My Favorites</button>
+        <label for='view-to-cook-button' class='hidden'>My Recipes</label>
+        <button id='view-to-cook-button' class='view-to-cook navButton'>My Recipes</button>
+        <form class='search-bar'>
         <label for='search-button' class='hidden'>Click here to search here</label>
```
Comment (gaj23) [inline src/index.html:22]: I think we can start applying the search functionality now, too.

File: src/scripts.js [hunk 3]
```diff
@@ -1,153 +1,157 @@
...
-    let userId = (Math.floor(Math.random() * 49) + 1)
-    let newUser = users.find(user => {
-        return user.id === Number(userId);
-    });
-    user = new User(userId, newUser.name, newUser.pantry)
-    pantry = new Pantry(newUser.pantry)
-    populateCards(cookbook.recipes);
-    greetUser();
+
+function loadData() {
```
Comment (gaj23) [inline src/scripts.js:27]: I appreciate the renaming of this function! Thank you.

**Review by gaj23 — COMMENTED** [review]
I agree with Max about some refactoring, but everything looks good. I'll be pulling and checking things out this morning, too.

File: src/scripts.js [hunk 4]
```diff
@@ -1,153 +1,157 @@
...
-        </button>
-        <label for='favorite-button' class='hidden'>Click to favorite recipe
-        </label>
-        <button id='${recipe.id}' aria-label='favorite-button' class='favorite favorite${recipe.id} card-button'></button>
-      </header>
-        <span id='${recipe.id}' class='recipe-name'>${recipe.name}</span>
-        <img id='${recipe.id}' tabindex='0' class='card-picture'
-        src='${recipe.image}' alt='click to view recipe for ${recipe.name}'>
-  </div>`)
+function displaySearchRecipes(event) {
```
Comment (gaj23) [inline src/scripts.js:148]: I noticed that this functionality wasn't working when I tried it on my local device.

**Pull request merged**